app/models/user.rb

```diff
@@ -410,6 +410,14 @@ def search_by_username(query)
       User.where('MATCH(username) AGAINST(? IN BOOLEAN MODE)', "#{query}")
     end
 
+    def search_people(query)
+      if query
+        User.where('username LIKE ?', "%#{query}%")
```

Comment:
I'm just a little worried about the performance of this query, as our user table is really huge due to spam users. I'm going to suggest some refinements as a suggestion, and then I think we can merge this! Starting with, only using wildcards on the end of the search query, to make a more optimized db search query. Then, filtering by user status! Thanks!

```suggestion
        User.where('username LIKE ?', "#{query}%")
          .where(status: 1)
```

Reply:
Okay, sounds good to me! Thanks for the review